### PR TITLE
Emails chasing teams for details for sending their kits

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This adds three emails for teams who have submitted varying levels of information:

- those who we've not heard from
- those who submitted just a disclaimer (via email)
- those who submitted just contact & address details (web form)

### CODE REVIEW

File: SR2024/2023-09-27-chase-details-for-kits.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+We hope you're as excited about the start of the Student Robotics year as we
+are!
+
+As you're hopefully aware, the year starts with a virtual [Kickstart][kickstart]
+event on Saturday 21st October which gives competitors and team supervisors an
+introduction to this year's competition and overview of how to build and
+engineer their robot.
+
+Ahead of Kickstart we'd planning to send out your loaned [robotics kit][kit-docs],
```
Comment (RealOrangeOne) [inline SR2024/2023-09-27-chase-details-for-kits.md:18]:
```suggestion
Ahead of Kickstart we're planning to send out your loaned [robotics kit][kit-docs],
```

File: SR2024/2023-09-27-chase-details-for-kits.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+your secondary contact:
+
+    https://forms.gle/jgJ6fhspWUNyWR5X9
+
+The shipping address we have for you is:
+
+    SHIPPING_ADDRESS
+
+To ensure that you receive your robotics kit before Kickstart, please provide
+us with the details necessary by Sunday 1st October.
```
Comment (RealOrangeOne) [inline SR2024/2023-09-27-chase-details-for-kits.md:43]: Question: Is it intentional that the date is duplicated here? (I assume yes for emphasis)

File: SR2024/2023-09-27-chase-kit-disclaimers-only.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+attachments:
+  - Kit Disclaimer (PDF): https://drive.google.com/file/d/1a4Uh3O5qYElmA_QPztpVyvAByUGf4T3H/view
+---
+
+Hello,
+
+We hope you're as excited about the start of the Student Robotics year as we
+are!
+
+Ahead of Kickstart we'd planning to send out your loaned [robotics kit][kit-docs],
```
Comment (RealOrangeOne) [inline SR2024/2023-09-27-chase-kit-disclaimers-only.md:17]:
```suggestion
Ahead of Kickstart we're planning to send out your loaned [robotics kit][kit-docs],
```

File: SR2024/2023-09-27-chase-secondary-contacts-and-shipping-address.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+   - not confirmed their shipping address
+subject: "Action Required: Receiving your kit for Student Robotics 2024"
+---
+
+Hello,
+
+We hope you're as excited about the start of the Student Robotics year as we
+are!
+
+Ahead of Kickstart we'd planning to send out your loaned [robotics kit][kit-docs],
```
Comment (RealOrangeOne) [inline SR2024/2023-09-27-chase-secondary-contacts-and-shipping-address.md:15]:
```suggestion
Ahead of Kickstart we're planning to send out your loaned [robotics kit][kit-docs],
```

File: SR2024/2023-09-27-chase-details-for-kits.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2024/2023-09-27-chase-details-for-kits.md:43]: Yeah, it was duplicated in the place confirmation email too.

File: SR2024/2023-09-27-chase-details-for-kits.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2024/2023-09-27-chase-details-for-kits.md:18]: 022dc2c1f1965abbd82bdeb7ad1ca70af1f5c4f9

File: SR2024/2023-09-27-chase-kit-disclaimers-only.md (hunk 3 above)
Comment (PeterJCLaw) [reply SR2024/2023-09-27-chase-kit-disclaimers-only.md]: 022dc2c1f1965abbd82bdeb7ad1ca70af1f5c4f9

File: SR2024/2023-09-27-chase-secondary-contacts-and-shipping-address.md (hunk 4 above)
Comment (PeterJCLaw) [reply SR2024/2023-09-27-chase-secondary-contacts-and-shipping-address.md]: 022dc2c1f1965abbd82bdeb7ad1ca70af1f5c4f9

**Review by RealOrangeOne — APPROVED** [review]

**Review by Adimote — APPROVED** [review]

**Pull request merged**